src/general/semidiscretization.jl

```diff
@@ -248,7 +248,7 @@ timespan: (0.0, 1.0)
 u0: ([...], [...]) *this line is ignored by filter*
 ```
 """
-function semidiscretize(semi, tspan; reset_threads=true)
+function semidiscretize(semi, tspan; reset_threads=true, data_type=nothing)
```

Comment:
fix doc string

Reply:
I'll add this in #493 when this feature is actually working.